wrapper.js

```diff
@@ -61,6 +79,8 @@ function setupMethods (soljson) {
   var wrapCallbackWithKind = function (callback) {
     assert(typeof callback === 'function', 'Invalid callback specified.');
     return function (context, kind, data, contents, error) {
+      // Must be a null pointer.
```

Comment:
Why? Just now?

Reply:
It was an oversight. It helps catching cases when the API changed but we didn't update it here. Otherwise it doesn't do anything.